Gemfile

```
@@ -4,8 +4,9 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
-
+gem 'discard', '~> 1.2'
```

Esta gema deberías agregarla en la linea 11 debajo de bootsnap, en el mismo formato de las demas:
```suggestion
gem 'discard', '~> 1.2' # https://rubygems.org /gems/discard
```